wiki/mapping/glossary.md

```diff
@@ -16,232 +16,242 @@ let us know in #mapping-discussion!
+| **Inline** | A [Pattern](#p) that remains in one [Column](#c) and [Row](#r) for two or more [Blocks](#b) in sequence. This typically refers to two alternating colors, but not always. <details><summary>**Example Image**</summary>![Picture of inline](~@images/mapping/inline.png)</details> |
+| **Inner Ring** | A small ring around the [Track](#t) found in some in-game [Environments](#e). |
+| **Invert** | [Blocks](#b) pointing inwards from the outside requiring a larger pre-swing to hit. <details><summary>**Example Image**</summary>![Picture of invert](~@images/mapping/invert.png)</details> |
 
 ## JK
 | Term&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Definition |
 | - | :- |
-| **.json File** | The text file containing most content for that difficulty or song info in the old 1.0 song format. Contains same format but different structure to .dat files, with a different file extension. File format is used for both song info and for map content. See also [Info File](#i), [Difficulty File](#d) |
-| **JSON Wall** | A style of mapping requiring Mapping Extensions in which the mapper can be extremely creative with non-standard wall types and sizes. Not recommended for novice mappers. |
-| **Jump** | A pattern that moves across multiple lanes/columns (horizontally) or layers/rows (vertically) in rapid succession. Not recommended below 1/2 precision, especially at high tempo. The faster the jumps are, the more difficult they are to execute. See also [Jump Stream](#jk) <details><summary>**Example Image**</summary>![Picture of jump](~@images/mapping/jump.jpg)</details>|
-| **Jump Stream** | A style of mapping that includes jumps within a stream. <details><summary>**Example Image**</summary>![Picture of jump stream](~@images/mapping/jumpstream.jpg)</details>|
+| **.json File** | The text file containing most content for a certain difficulty or song info in the old 1.0 song format. It contains JSON format with a different structure to [.dat Files](#d), and a different file extension. This file format is used for both song info and for map content. See also: [Difficulty File](#d), [Info File](#i). |
+| **JSON Wall** | A style of mapping requiring the mod [Mapping Extensions](#m), in which the mapper can be extremely creative with non-standard [Wall](#wxyz) types and sizes. Not recommended for novice mappers. |
+| **Jump** | A [Pattern](#p) that moves across multiple [Columns](#l) horizontally or [Rows](#r) vertically in rapid succession. This pattern is not recommended below 1/2 [Precision](#p), especially at high [Tempos](#t). The faster the jumps are, the more difficult they are to execute. See also: [Jump Stream](#jk). <details><summary>**Example Image**</summary>![Picture of jump](~@images/mapping/jump.jpg)</details> |
```

Comment:
Columns should link to \#c.

> This pattern is not recommended below 1/2 [Precision](#p), especially at high [Tempos](#t). The faster the jumps are, the more difficult they are to execute.

After reading this again, this part seems pretty dated now. Might be worth removing.

Reply:
Agreed, the definition feels more precise without the added difficulty warning as well.